tests/Makefile

```diff
@@ -16,6 +16,7 @@ BINS=\
 	getid \
 	link \
 	math \
+	mem \
```

Comment:
mem must also be added to the .gitignore

Reply:
no worries